frontend/src/types/Section.ts

```diff
@@ -0,0 +1,5 @@
+export default interface Section {
+    id: number;
+    group: string;
```

Comment:
The group prop can be removed, here and in the backend serialization. It was added when the evaluation of the matching pairs logic was done in the frontend. Now that this is moved to backend, the gorup prop is not needed anymore.